src/index.js

```diff
@@ -81,7 +82,10 @@ export const Webform = ({
   });
 
   useEffect(() => {
-    setUiText(locale?.[lang] || locale.en);
+    const UILocale = locale?.[lang] || locale.en;
+    const UITextParam = UIText?.[lang] || {};
+    setUiText({ ...UILocale, ...UITextParam });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Are there any alternative for this? maybe `useMemo` or `useCallback` @wayangalihpratama ?

Reply:
Pushed an update mas @dedenbangkit

Reply:
Mantap